frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -969,6 +970,18 @@ export const ClusterSettingsAlerts: React.FC<ClusterSettingsAlertsProps> = ({
   const pausedMCPs = machineConfigPools
     .filter((mcp) => !isMCPMaster(mcp))
     .filter((mcp) => isMCPPaused(mcp));
+  if (!canUpgrade) {
+    return (
+      <Alert
+        variant="info"
+        isInline
+        title={t(
+          'public~Control plane is externally managed.', // TODO: final copy
+        )}
+        className="co-alert"
```

Comment:
@sg00dwin, @mattnolting is `co-alert` on y'all's radar as you are working through the dark theme work and removing overrides?  Seems like it should be if it's not.  Yea/nay?

Reply:
> Any reason we don't lean fully into PF? I get it this is sorta our standard for the admin side, but I think we should look at why we can't use PF directly and without a class.... if it's missing something, we should address that in PF.

One significant thing `co-alert` does is provide `margin-bottom`.  Without it, alerts don't get `margin-bottom` as PatternFly doesn't specify it.  But there is a Bootstrap override that is no longer relevant.

Reply:
Recommend you lean into the layouts that PF provides. `Stack` is a good one for auto gapping items.

Reply:
A fair question Robb! Why have 2-ish more components when we could just use a css className + a css variable.

Well... we are using a component library to handle our entire app's styles... and if we did it from the ground up (and PF 4 was available in it's current state at that time), we'd never have to worry about what if the spacing is too much, what if they are in mobile and the spacing needs to auto shrink across our entire app because of the limited view space... what about dark theme, what about accessibility, what about etc... The style of our app would be entirely controlled by PF and the UX team. Thus not needing individual adjustments or "class name knowledge" to make identical structured UIs. Everything is just PF and its accessible docs.

Leaning 100% into your component library, especially one as comprehensive as PF, allows you to stop fiddling with your custom values; spacing, colours, sizes of text, other accessibility concerns. Sure, there are PF css variables... but mostly they exist for non-React apps.

Reply:
I understand your point.  But adding extra markup to simply control the layout is *more* code, which increases the maintenance complexity.  Searching for a classname across a repo is easier than searching for nested components.  If your advocating for less custom rules, I get that.  Perhaps `<Alert className="pf-u-*" ... />`  is the middle ground?

Reply:
I truly appreciate this conversation!  Given the other alerts within this component make use of `co-alert` and we have a number of other instances across the code base, I'm inclined to leave this as is and circle back to it later.  IMO, it's better to be consistently wrong than inconsistently right.

Reply:
That's fair Robb.... I retract my request. Please keep it the way you have it. We'll probably need to discuss as a team various improvements to look out... using custom CSS to do PF things is not ideal. Even if you lean into the PF css variables.

Reply:
> We'll probably need to discuss as a team various improvements to look out

Yes!  Given the history of where we started with this code base and where it is headed, it would be beneficial to make sure everyone has a shared understanding of best practices since they are evolving.  :)